tools/make-srpm

```diff
@@ -26,45 +26,9 @@ fi
 base=$(cd $(dirname $0); pwd -P)
 rpm_tmpdir=$(mktemp -d -t rpmbuild.XXXXXX)
 
-# First make a .tar.gz with the source code
-(
-    cd $base/..
+srpm=$(cd ${rpm_tmpdir} && "$base/../test/guest/lib/make-srpm" $("$base/make-tar"))
```

Comment:
I would suggest disentangling this a bit, and calling make-tar (or make-source) on a prior line.